Source/ProgressionSystemRuntime/Public/Data/PSTypes.h

```diff
@@ -60,4 +60,8 @@ struct FPSRowData : public FTableRowBase
 	/** Defines if level is locked or not */
 	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category="C++")
 	bool IsLevelLocked = true;
+
+	/** Defines the star animation for each character */
```

Comment:
There are might be multiple animations in this row, please mention this animation is called on in-game cinematic started

Reply:
updated as part of https://github.com/h4rdmol/ProgressionSystem/pull/12/commits/82d67ef9c4cfefaf80d1e43ecfa1acc873db5dac